src/core/client/stream/tabs/Live/LiveComment/LiveCommentContainer.css

```diff
@@ -44,6 +77,14 @@
   height: 20px;
 }
 
+.parentUserHighlight {
+  font-weight: var(--font-weight-primary-semi-bold);
+  color: var(--palette-text-000);
+
+  margin-right: var(--spacing-2);
+  height: 20px;
```

Comment:
Don't use `height: 20px` here on this typography element. Did you mean `line-height` instead?